modules/bluefish/consumer/bluefish_consumer.cpp

```diff
@@ -127,7 +127,7 @@ bool get_videooutput_channel_routing_info_from_streamid(bluefish_hardware_output
 
 struct bluefish_consumer : boost::noncopyable
 {
-	spl::shared_ptr<BvcWrapper>							blue_;
+	spl::shared_ptr<bvc_wrapper>							blue_;
```

Comment:
Maybe this should be held by value instead which would also remove the need for `create_blue()` entirely.